packages/dds/sequence/src/sequence.ts

```diff
@@ -243,7 +243,12 @@ export abstract class SharedSegmentSequence<T extends MergeTree.ISegment>
         return this.client.getLength();
     }
 
-    public getPosition(segment): number {
+    /**
```

Comment:
You can likely use @inheritDoc format here to refer to client API

Reply:
I don't think @inheritdoc  works with composition

Reply:
As far as I can see, it does not care. You just point to any symbol